dotenv.example

```diff
@@ -115,3 +115,12 @@
 # Speeds up turn around times of tests
 # OSEM_FACTORY_LINT=false
 
+# How should browser tests be performed?
```

Comment:
Why does this need to be configurable?

Reply:
_I_ want to test on Chrome _and_ Firefox, but I don't see the need to run the whole suite on both browsers all the time, so, it's configurable. In this way, I can either

```
rspec spec/features
````

to just run the suite... or 

```
OSEM_TEST_DRIVER=firefox rspec spec/features
```

to, for example, see it in Firefox. Makes me happy.

Reply:
>  Makes me happy.

If that's the case, ship it! :heart_eyes: